src/components/home page components/HomePageAnnouncementList.js

```diff
@@ -12,16 +13,19 @@ const HomePageAnnouncementList = () => {
 
   today = dd + '/' + mm + '/' + yyyy;
   let time = HH + ":" + MM;
+  
+
   return(
     <Card className='mt-3'>
       <Card.Body>
         <Card.Title>
           Известувања
         </Card.Title>
         <ListGroup>
-          <HomePageAnnouncement text="Итно известување" variant="danger"/>
-          <HomePageAnnouncement text="Предупредување" variant="warning"/>
-          <HomePageAnnouncement text="Информативно известување" variant="info"/>
+          {announcements.map(announcement => 
```

Comment:
We need variant in order to determine the importance of the warning, so we need text and variant, either impl in data file or hardcode variant="danger/warning/info"